src/forms/CheckboxGroup.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react"
+
+import "./CheckboxGroup.scss"
+import "../actions/Button.scss"
+import { ButtonProps } from "actions/Button"
+
+export interface CheckboxGroupProps extends Pick<ButtonProps, "size" | "variant"> {
+  /** Label content */
+  label?: string
+  /** Current selected values*/
+  values: string[]
+  /** An array of strings representing each item in the group*/
+  options: string[]
```

Comment:
I would recommend having separated text labels + values, as sometimes values might need to be machine-readable and consistent but the labels would be human-readable/translated./etc.